docs/source/user/accessibility.rst

```diff
@@ -0,0 +1,130 @@
+
+-----------------------------------
+Keyboard Navigation
+-----------------------------------
+
+Keyboard navigation lies at the core of JupyterLab's accessibility approach, allowing users to efficiently interact with the interface using keystrokes. It has naturally evolved to provide a quicker alternative to mouse-based interactions by the use of multiple accessibility tools throughout the interface that will be discussed in this section.
+
+-----------------------------------
+Keyboard Shortcuts
+-----------------------------------
+
+JupyterLab offers an extensive set of keyboard shortcuts that empower users to navigate through various functions and tools. From executing code cells to managing tabs, these shortcuts streamline workflows and enhance accessibility for users who rely on keyboard input.
+
+For a comprehensive list of available shortcuts, users can explore the advanced settings by navigating to `Settings > Advanced Settings` in the JupyterLab main menu.
+
+.. figure:: ./desktop/quansight_labs/documentation/keyboard_shortcut_final.png
+   :alt: an image showing how to add a shortcut in the advanced settings panel.
+
+Within JupyterLab's Advanced Settings, default keyboard shortcuts are unmodifiable, though users can craft and modify their own shortcuts to align with their unique workflows. In order to create a new keyboard shortcut use the `add` button and enter the information requested,
+Please note that the information can be removed and edited afterwards.
+
+-----------------------------------
+Command Palette
+-----------------------------------
+
+The Command Palette acts as a command center, offering a text-based interface for executing commands. It can be accessed from the `View` menu or using the keyboard shortcut `Ctrl + Shift + C`. This feature not only aids keyboard-centric users but also promotes efficiency and
+accessibility, as users can swiftly access functionalities without navigating complex menus.
+
+
+.. figure:: ./desktop/quansight_labs/documentation/command_palette.png
+   :alt: an image showing the activated command palette modal.
+
+-----------------------------------
+Screen Reader Support
+-----------------------------------
+
+While JupyterLab has taken strides to support screen reader compatibility, it's important to note that there hasn't been any extensive testing of this feature. Presently, screen readers such as `NVDA <https://www.nvaccess.org/download/>`_ and `JAWS <https://www.freedomscientific.com/products/software/jaws/>`_ on Windows, as well as `VoiceOver <https://support.apple.com/en-ng/guide/voiceover/vo2682/mac#:~:text=You%20can%20also%20turn%20VoiceOver,then%20press%20the%20Space%20bar.>`_ on macOS, provide some level of compatibility. However, challenges exist, such as reading images without labels, equations, and links.
```

Comment:
```suggestion
While JupyterLab has taken strides to support screen reader compatibility, it's important to note that there hasn't been any extensive testing of this feature. Presently, screen readers such as `NVDA <https://www.nvaccess.org/download/>`_ and `JAWS <https://www.freedomscientific.com/products/software/jaws/>`_ on Windows, as well as `VoiceOver <https://support.apple.com/en-ng/guide/voiceover/welcome/mac>`_ on macOS, provide some level of compatibility. However, challenges exist, such as reading images without labels, equations, and links.
```

Suggestion to avoid linking to any specific version, and including the highlight text